test/rest.test.js

```diff
@@ -1571,6 +1571,41 @@ describe('strong-remoting-rest', function() {
           .end(expectErrorResponseContaining({ message: 'an error' }, done));
       });
 
+      it('should return 500 if an array of errors is thrown', function(done) {
+        var duplicateError = new Error('duplicate error');
+        var errArray = [duplicateError, duplicateError];
+
+        var expectedMessage = 'Failed with multiple errors, see `details` for more information.';
+
+        function method(error) {
+          return givenSharedStaticMethod(function(cb) {
+            cb(error);
+          });
+        }
+
+        request(app).get(method(duplicateError).url)
+          .set('Accept', 'application/json')
+          .expect(500)
+          .end(function(err, res) {
+            if (err) return done(err);
+            expectedDetail = res.body.error;
+            delete expectedDetail.statusCode;
+
+            request(app).get(method(errArray).url)
+            .set('Accept', 'application/json')
+            .expect(500)
+            .end(function(err, res) {
+              if (err) return done(err);
+              console.log(res.body.error);
+              var error = res.body.error;
+              expect(error).to.have.property('message', expectedMessage);
```

Comment:
This is brittle. If we decide to reword the message in the future, this assertion will fail.

I think this should be little bit more robust, while it should still fail with the current implementation on master:

```js
expect(error).to.have.property('message').that.match(/multiple errors/);
```